dkms.in

```diff
@@ -3405,10 +3409,18 @@ case "$action" in
             check_rw_dkms_tree
         fi
         "${action}_module"
+        # Execute post_transaction command if set
+        if [[ -n "$post_transaction" ]] && [[ $action = install ]]; then
```

Comment:
What about uninstall? I would suggest adding a second argument that is "1" when installing or "0" when removing. Or maybe avoid these kinds of [ambiguous conventions](https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax) from RPM packages for example and just use strings that can be matched easily in case statements. E.g. "install" "update" "remove"?

Reply:
It's already set for `uninstall`. It's set for `remove/unbuild/uninstall`, `install` and `autoinstall`; so tied any command which result in modules being changed in the kernel directory. All the other commands (`add` etc.) happens without performing any change in the kernel directory so the command is not executed there.

https://github.com/dell/dkms/blob/post-action/dkms.in#L3388-L3423